api/src/server.ts

```diff
@@ -28,6 +29,11 @@ app.use(express.json({ limit: "10mb" }));
 app.get("/api/terms/:termId/courses/:courseId", notangles.getCourse);
 app.get("/api/terms/:termId/courses", notangles.getCourseList);
 app.get("/api/terms/:termId/freerooms", freerooms.getFreeroomsData);
+app.get("/api/startdate/freerooms", getStartDateFreerooms);
+
+app.get("/api/startdate/notangles", getStartDateNotangles);
```

Comment:
Can you follow the other endpoints and have the Notangles/Freerooms specific functions in the Notangles/Freerooms file? Then maybe add another folder for files containing helper/shared functions like getStartDate.ts

Reply:
Will finish this later tonight!

Reply:
I have added this functionality, would love feedback on it!